javascript/javascript2/week1/hyf-bay/main.js

```diff
@@ -0,0 +1,24 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+
+function renderProducts(products) {
+    for (let i = 0; i <= products.length - 1; i++) {
+        //Select ul
+        const myProductList = document.querySelector(".product-list")
+
+        //Create li
+        const list = document.createElement("li");
```

Comment:
Confusing variable name. This is a list item not a list.

Reply:
Changed the variable name